spec/parser_spec.lua

```diff
@@ -33,9 +33,9 @@ end
 -- Assertions for full programs
 --
 
-local function parse(program_str)
-    assert(util.set_file_contents("__test__.pln", program_str))
-    return driver.compile_internal("__test__.pln", "ast")
+local function parse(code)
+    assert(util.set_file_contents("__test__.pln", code))
+    return driver.compile_internal("__test__.pln", code, "ast")
```

Comment:
Can we get away with removing the call to `set_file_contents`?

Reply:
I thought we were doing in this in a different PR. :/

Reply:
Sure. We can do it in another PR. There is already an issue for it so we won't forget it.